### PR TITLE
CY-1795 - Update Manager Service Status to use new /status REST API

### PR DESCRIPTION
**Pull request by qooban**
### Before
image
image

### After
image
image

### CODE REVIEW

File: app/actions/status.js [hunk 1]
```diff
@@ -32,9 +32,19 @@ export function getStatus() {
         return managerAccessor
             .doGet('/status')
             .then(data => {
-                let services = _.filter(data.services, item => !_.isEmpty(item.instances));
-                services = _.sortBy(services, service => service.display_name);
-                dispatch(setStatus(data.status, services));
+                const { services, status } = data;
+                const filteredServices = _.chain(services)
```
Comment (kubama) [inline app/actions/status.js]: FYI: `_.chain(...)` has a shorter equivalent: `_(...)`

File: app/actions/status.js [hunk 2]
```diff
@@ -32,9 +32,19 @@ export function getStatus() {
...
+                    .keys()
+                    .map(serviceName => ({
+                        name: serviceName,
+                        isExternal: services[serviceName].is_external,
+                        status: services[serviceName].status,
+                        description: _.get(services[serviceName], 'extra_info.systemd.instances[0].Description', '')
+                    }))
+                    .sortBy(service => service.name)
```
Comment (kubama) [inline app/actions/status.js]: I would do the `.sortBy` directly on service names array, before `.map`. It wouldn't be necessary to pass `service => service.name` as argument.

File: app/actions/status.js (hunk 1 above)
Comment (qooban) [reply app/actions/status.js]: 👍

File: app/actions/status.js (hunk 2 above)
Comment (qooban) [reply app/actions/status.js]: Good point. Refactored.

**Pull request merged**